### PR TITLE
Bookings

### PR DESCRIPTION
**Pull request by hlmgnc**

### CODE REVIEW

File: booking.js [hunk 1]
```diff
@@ -1,14 +1,17 @@
...
+    constructor(client, barber, date) { 
+        this.client = client       
+        this.barber = barber        
+        this.date = date
+        const bookingId = uuidv4();
+        this.bookingId = bookingId
             //console.log(bookingId)
         }
+
+        addBooking(booking) {
```
Comment (dashersw) [inline booking.js:12]: why do you have this `addBooking` function here?

File: index.js [hunk 2]
```diff
@@ -17,25 +17,27 @@
...
-  console.log(bookings)
+  //console.log(bookings)
   //console.log("UPDATE"); 
-  apptDate.setMonth(7); 
+  //apptDate.setMonth(7); 
   //hilmi.updateBooking(cem, apptDate); 
-  console.log(gaby.bookings);
-  console.log(furkan.bookings);
+  console.log(gaby.bookings); 
+  //console.log(furkan.bookings);
```
Comment (dashersw) [inline index.js:40]: it's best to remove unused console.logs from your code.

Comment (hlmgnc) [reply index.js]: I removed the unused console.logs from my code.

File: booking.js (hunk 1 above)
Comment (hlmgnc) [reply booking.js]: I removed the addBooking function and created a new booking function.

**Pull request merged**